packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -57,21 +77,28 @@ import TeraDrilldownHeader from '@/components/drilldown/tera-drilldown-header.vu
 import { TabViewChangeEvent } from 'primevue/tabview';
 import { computed, ref, useSlots } from 'vue';
 import TeraColumnarPanel from '@/components/widgets/tera-columnar-panel.vue';
-import { WorkflowNode } from '@/types/workflow';
+import { WorkflowNode, WorkflowOutput } from '@/types/workflow';
 import TeraOperatorAnnotation from '@/components/operator/tera-operator-annotation.vue';
 import Chip from 'primevue/chip';
 import TeraOperatorPortIcon from '@/components/operator/tera-operator-port-icon.vue';
+import { isEmpty } from 'lodash';
+import Menu from 'primevue/menu';
+import Button from 'primevue/button';
+import TeraOutputDropdown from '@/components/drilldown/tera-output-dropdown.vue';
 
 const props = defineProps<{
 	node: WorkflowNode<any>;
+	menuItems?: any[];
 	title?: string;
 	tooltip?: string;
 	popover?: boolean;
+	drillDownOutputs?: WorkflowOutput<any>['id'];
+	drillDownOptions?: WorkflowOutput<any>[] | { label: string; items: WorkflowOutput<any>[] }[];
```

Comment:
Drilldown context is understood
```suggestion
	output?: WorkflowOutput<any>['id'];
	outputOptions?: WorkflowOutput<any>[] | { label: string; items: WorkflowOutput<any>[] }[];
```